Dockerfile

```diff
@@ -1,21 +1,28 @@
 # Use an official Python runtime as a parent image
 FROM python:3.6-slim
 
-ENV PYTHONUNBUFFERED 1
-
-ENV NODE_ENV production
-
 # add our user and group first to make sure their IDs get assigned consistently
 RUN groupadd -r zeus && useradd -r -m -g zeus zeus
 
-RUN mkdir -p /usr/src/app
-WORKDIR /usr/src/app
+ENV PYTHONUNBUFFERED 1
+
+# Sane defaults for pip
+ENV PIP_NO_CACHE_DIR off
+ENV PIP_DISABLE_PIP_VERSION_CHECK on
+
+RUN mkdir -p /usr/src/zeus
+WORKDIR /usr/src/zeus
 
 RUN set -x \
-    && apt-get -qy update \
-    && apt-get -qy install -y --no-install-recommends \
```

Comment:
Not sure why you were installing system python3 again, but that's not needed since it already exists.

Also, it's important to make sure to follow up these layers with `rm -rf /var/lib/apt/lists/*` to remove the apt cache from disk before the layer is finalized.